troposphere/static/js/components/projects/resources/instance/details/sections/details/CreatedFrom.jsx

```diff
@@ -15,8 +15,10 @@ export default React.createClass({
     },
 
     render: function() {
-        var instance = this.props.instance,
-            image = stores.ImageStore.get(instance.get("image").id);
+        let { instance } = this.props,
+            image = instance ?
+                    stores.ImageStore.get(instance.get("image").id)
```

Comment:
Let's break this out, because there are already situations where `instance.get('image')` could be `{}` or `null`